src/core/config/styles.js

```diff
@@ -1,22 +1,37 @@
+import get from "lodash/get";
 import { createMuiTheme } from '@material-ui/core/styles';
-
-import backgroundImage from "../../version/images/Artboard.png";
-import cardBackgroundImage from "../../version/images/blue-ring-01.png";
-
-import { MAIN_THEME_COLOR } from "../../version/config/theme.config";
+import { themeImages } from "../../version/config/theme.config";
 
 export const ITEMS_PER_PAGE = 10;
 
-const CORE_MAIN_COLOR = "#0D672F";
+const DEFAULT_COLOR = "#0D672F";
+
+/**
+ * This function defines theme color
+ *
+ * @author Bogdan Shcherban <bsc@piogroup.net>
+ * @param {boolean} isContrastMode
+ * @return {string}
+ */
+export function getCurrentThemeColor(isContrastMode = false) {
+    let currentThemeColor = (window && window.config) ? window.config.mainColor : DEFAULT_COLOR;
+    return (isContrastMode) ? "#000" : currentThemeColor;
```

Comment:
I think the config should be able to specify the contrast mode color as well. So I'd write something like:

```js
 return (isContrastMode)
   ? (window && window.config) ? window.config.mainContrastColor : CONTRAST_COLOR
   : (window && window.config) ? window.config.mainColor : DEFAULT_COLOR;
```

But material ui has a support for themes as well, so I advise you to use those. It's well thought and IMHO offers everything you need in terms of customization.

https://material-ui.com/customization/themes/#type-light-dark-theme

Reply:
Can I used the first solution temporary?
Could we provide Theme via Material UI later? Because right now we should start work with Scotland version - this issue has more priority.

Reply:
Done